flask_rest_paginate/__init__.py

```diff
@@ -60,8 +59,12 @@ def _is_integer(num):
             size = self._default_page_size
 
         # TODO: linked to db exception
-        # if Model, run the default query
-        if isinstance(query_model, type(self._db.Model)):
+        # if model is a list
+        if isinstance(query_model, list):
+            from flask_rest_paginate.list_pagination import paginate, ListPagination
```

Comment:
Remove this unused import

```suggestion
            from flask_rest_paginate.list_pagination import paginate
```